lib/models/material_colors.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+const MaterialColor accentColor =
+    const MaterialColor(0xFF969696, const <int, Color>{
+  50: Color.fromRGBO(150, 150, 150, .1),
+  100: Color.fromRGBO(150, 150, 150, .2),
+  200: Color.fromRGBO(150, 150, 150, .3),
+  300: Color.fromRGBO(150, 150, 150, .4),
+  400: Color.fromRGBO(150, 150, 150, .5),
+  500: Color.fromRGBO(150, 150, 150, .6),
+  600: Color.fromRGBO(150, 150, 150, .7),
+  700: Color.fromRGBO(150, 150, 150, .8),
+  800: Color.fromRGBO(150, 150, 150, .9),
+  900: Color.fromRGBO(150, 150, 150, 1),
```

Comment:
I would think that when using `MaterialColor` - we would already have a whole palette without the need to define all of this.